httpd.go

```diff
@@ -61,3 +78,11 @@ func messageHandler(p Param) (int, string) {
 		return 200, fmt.Sprintf("Message sent successfully to %s", p.Channel)
 	}
 }
+
+func sendAsync(p Param, postTime time.Time, ch chan error) (int, string) {
```

Comment:
Thank you for review.

> it doesn't send message asynchronously

Indeed. will change `sendNow()` and `sendLater()` respectively.